pages/packages/gradle.md

```diff
@@ -49,18 +49,80 @@ The following steps describe the process above:
       repositories {
         maven {
           // Define the Buildkite repository to publish to
-          url "https://buildkitepackages.com/{org.slug}/{java.gradle.package.repository.name}/java/maven2/"
+          url "https://buildkitepackages.com/{org.slug}/{java.package.repository.name}/java/maven2/"
           credentials {
-            username = "java-gradle-package-repository-credentials"
+            username = "java-package-repository-credentials"
             password = ""
           }
         }
       }
     }
     ```
 
+    where:
+    <%= render_markdown partial: 'packages/java_package_domain_name_version' %>
+
+    <%= render_markdown partial: 'packages/org_slug' %>
+
+    <%= render_markdown partial: 'packages/java_package_repository_name' %>
+
 1. Publish your package:
 
     ```bash
     gradle publish
     ```
+
+## Access a package's details
+
+<%= render_markdown partial: 'packages/access_java_package_details_page' %>
+
+<%= render_markdown partial: 'packages/package_details_page_sections' %>
+
+### Downloading a package
+
+A Java (Jar) package can be downloaded from the package's details page.
+
+To download a package:
+
+1. [Access the package's details](#access-a-packages-details).
+1. Select _Download_.
+
+### Installing a package
+
+1. [Access the package's details](#access-a-packages-details).
+1. Ensure the _Installation_ > _Installation instructions_ section is displayed.
+1. Copy the code snippet and paste this into the `build.gradle` Gradle file to install this package.
+
+The package code snippet is based on this format:
+
+```gradle
+// Add to 'build.gradle' file
+
+// You require these plugins
+// "java": To work with java libraries
+['java' ].each {
+  apply plugin : it
+}
+
+// maven type repository to download/install java packages from
+repositories {
+  maven {
+    url "https://buildkitepackages.com/priv/{api.token}/{org.slug}/{java.package.repository.name}/maven2/"
```

Comment:
As mentioned, I don't think we should be supplying `/priv/{token}` urls, but we need to fix the in product.

Reply:
Thanks @sj26 , I've removed this and will await any further instructions required on how the API read token is configured for installing Gradle packages.

Reply:
Also, will need to know if the way we're passing the 'package repository write token' to publish Gradle-generated Java packages is correct in [this code snippet in the docs](https://2721--bk-docs-preview.netlify.app/docs/packages/gradle#publish-a-package).